controllers/C_Document.class.php

```diff
@@ -742,7 +742,7 @@ function retrieve_action($patient_id = "", $document_id, $as_file = true, $origi
                     }
                 } else {
                     $filetext = '';
-                    error_log("ERROR: Document '" . basename_international($url) . "' cannot be converted to JPEG. Perhaps ImageMagick is not installed?");
+                    error_log("ERROR: Document '" . errorLogEscape(basename_international($url))) . "' cannot be converted to JPEG. Perhaps ImageMagick is not installed?");
```

Comment:
This will cause a parsing error since you have a extra parenthesis. This should be:
```
error_log("ERROR: Document '" . errorLogEscape(basename_international($url)) . "' cannot be converted to JPEG. Perhaps ImageMagick is not installed?");
```